hack/lib/golang.sh

```diff
@@ -728,7 +728,7 @@ kube::golang::is_instrumented_package() {
 # Echos the path to a dummy test used for coverage information.
```

Comment:
We should replace this with `go build -cover`
`/assign`

Reply:
actually, I'm not sure this is worth it. the results will be incompatible and we're not heavily invested here, and last time this needed fixing with the current approach it was pretty small.